tests/merge_sorted_lists_test.py

```diff
@@ -1,2 +1,38 @@
 import pytest
 from functions.merge_sorted_lists import merge_sorted_lists
+
+
+def test_for_1_and_2():
```

Comment:
All tests are passing!

Reply:
Tests are written correctly and all are passing.